### ISSUE TITLE
Block colors are no good for color-blind people

### ISSUE THREAD

**Issue by Kenny2github** [LLK/scratch-blocks#1032]
### Context

I'm color blind - I'm a strong deutan - blues/purples and reds/greens/yellows are harder to distinguish

### Expected Behavior

I should be able to tell the differences between all the different block categories

### Actual Behavior

Motion, Looks, Sound look identical; so do Control, Events.

**Comment by towerofnix** [comment]
Maybe it would be best to keep discussion in #995?

**Comment by thisandagain** [comment]
Thanks for filing this @Kenny2github. This feels related but is a distinct issue from #995. A solution to one may solve the other, but this (and many other accessibility issues) is something we should be tracking more closely.

**Comment by Kenny2github** [comment]
As I did mention in #995, the reason this is an issue is because when I'm looking for blocks in one category, I frequently look in the wrong category - effectively doubling or even tripling my work times. And now there's three extra categories to confuse - so the colors need a change if colorblind people want to have normal work times

**Comment by Saskitan** [comment]
perhaps you could put symbals for each category

**Comment by Kenny2github** [comment]
@Saskitan that’s a good idea, actually, but the main problem is the color of the blocks when they’re in use - motion, looks and sound blocks, if stacked on top of each other, have no difference between them; same goes for control and events for 3.0

**Comment by mrjacobbloom** [comment]
#1051 talks about using icons in blocks to differentiate extensions--maybe that could apply to the default categories as well?

**Comment by Saskitan** [comment]
perhaps you could put symbals for each category on the blocks

**Comment by Saskitan** [comment]
not just the category windows

**Comment by marycps** [comment]
Completely agree. Scratch 3.0 version has poor contrast between the block background colors and the white Text.

If you use WebAim's color contrast checker at https://webaim.org/resources/contrastchecker
to check the contrast level between the mustard yellow Events blocks with the white text, for instance,
they fail the Level AA and Level AAA Accessibility checks.

**Comment by apple502j** [comment]
Adding icons (like extensions) on blocks is suggested (maybe with toggleable options)

**Comment by Kenny2github** [comment]
@apple502j I think @Saskitan suggested that nearly 3 years ago
> perhaps you could put symbals for each category on the blocks
